src/xcode/ENA/ENA/Source/Scenes/App/AppNavigationController.swift

```diff
@@ -179,34 +179,28 @@ class TabBarScrolling: NSObject, UITabBarControllerDelegate {
 	func tabBarController(_ tabBarController: UITabBarController, shouldSelect viewController: UIViewController) -> Bool {
 		if viewController == tabBarController.selectedViewController {
 			if let naviVC = viewController as? UINavigationController {
-				naviVC.scrollEmbeddedTableToTop()
+				naviVC.scrollEmbeddedViewToTop()
 			}
 		}
 		return true
 	}
 }
 
 extension UINavigationController {
-	// only if on top level of the navigation hierarchy: scroll the embedded table view up
-	func scrollEmbeddedTableToTop() {
+	// only if on top level of the navigation hierarchy: scroll the embedded view up
+	func scrollEmbeddedViewToTop() {
 		guard !isEditing && presentedViewController == nil && viewControllers.count == 1 else {
 			return
 		}
-		if let tableView = topViewController?.embeddedTableView {
-			scrollTableToTop(tableView)
+		if let embeddedScrollView = topViewController?.scrollView {
+			scrollViewToTop(embeddedScrollView)
 		}
 	}
 
-	private func scrollTableToTop(_ tableView: UITableView) {
-		guard !tableView.isEditing else {
-			return
+	private func scrollViewToTop(_ scrollView: UIScrollView) {
+		scrollView.scrollRectToVisible(CGRect(x: 0, y: 0, width: 1, height: 1), animated: true)	// no large title
+		DispatchQueue.main.asyncAfter(deadline: .now() + 0.3) {
```

Comment:
Where does the magic number come from? 🤔

Reply:
@marcussc indeed, a magic number. And if you activate slow motion in the simulator it will break ...

Apple's [Core Animation Programming Guide](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CoreAnimation_guide/AnimatableProperties/AnimatableProperties.html#//apple_ref/doc/uid/TP40004514-CH11-SW1) mentions 0.25 sec as duration for default property-based animations (CABasicAnimation) and default transition-based animations (CATransition); see tables B-2 and B-3. 
**Not sure what this means for a scrolling animation.** And to be on the safe side, I want slightly more than this duration; so I took the magic 0.3, and it seems to work, at least on the simulator. 
Note: I have not tested on a true device, although I am aware that this is strongly recommended.... but the damage in case of wrong timing is not so high.
If the time is too short it will stop at the first table row and not reveal the large title.
If the time is too long the animation will look less smooth.

*Do you know of any system constant or similar property that I could / should use instead?
Or should I add a comment with a short explanation?*

Oh, wait, I wrote a unit test ... if the time is indeed too short the unit test will break or be fragile ... maybe I should add a comment there...

Reply:
Yeah, add a comment, that would be nice

Reply:
Yeah a comment would be nice our you could implement the `scrollViewShouldScrollToTop(_:)` from `UIScrollViewDelegate` (https://developer.apple.com/documentation/uikit/uiscrollviewdelegate/1619382-scrollviewdidscrolltotop)

Reply:
@marcussc - UIScrollViewDelegate - cool idea 😎👍

Reply:
Unfortunately, my first tries on this really cool idea didn't work out as expected ....
I tried scrollViewDidEndScrollingAnimation, but on non-table scrollview it created a huge empty space below the title. For tableview it looked ok, only it did not reveal the large title if the table was already at the top row.

So I reverted to the previous solution (with the magic number) and only try to document this number better.

Reply:
Thanks for giving it a try! :)
With the new Comment it looks good to me.

Thank you again for your contribution.